CMakeLists.txt

```diff
@@ -725,7 +717,7 @@ int main(int argc, char *argv[]) {
     # This is kinda ugly but the PATH and HINTS don't get transmitted to sub-dependents
     set(CMAKE_SYSTEM_PREFIX_PATH_save ${CMAKE_SYSTEM_PREFIX_PATH})
     list(APPEND CMAKE_SYSTEM_PREFIX_PATH /opt/rocm)
```

Comment:
I don't think this explicit path should be here.

Reply:
Removing it would break the detection of the system install rocm (as installed from the RPMs distributed by AMD).

Reply:
Why cant you use PATHS or HINTS ? According to the cmake [documentation](https://cmake.org/cmake/help/latest/command/find_package.html) that's the preferred way.

Reply:
Cmake 'feature/bug': unlike ROCM_ROOT, they do not get propagated to the internal find_packages that are done within FindRocm and that results in an inconsistent/incomplete detection of internal features.